app/src/main/res/values/strings.xml

```diff
@@ -3,45 +3,45 @@
     <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
     <string name="data_management_label">Seedvault Backup</string>
-    <string name="current_destination_string">Backup status and settings</string>
+    <string name="current_destination_string">Back up status and settings</string>
```

Comment:
```suggestion
    <string name="current_destination_string">Status and settings for backups</string>
```
If not this, "backup" is a noun that is out of place. Translators are left guessing (as I did), and subsequently users for what to make of the translation.

Reply:
"Backup" as a noun seems like the correct usage here.